### DISCUSSION TITLE
Request to join

### DISCUSSION THREAD

**Discussion by ocharles** [haskell-github-trust/.github#6] [General]
Hello! :wave: I'm happy to maintain the various stuff we use at CircuitHub, and this org seems like a great idea. I'd be happy to join the trust if you'll have me!

**Comment by jamesdbrock** [comment]
Hi @ocharles sent you an invite!

**Comment by jamesdbrock** [comment]
@ocharles Keep in mind that if your repos are already in an actively maintained org like https://github.com/orgs/circuithub/repositories , then it's probably best to just keep them there. This org is more for repos that you are reluctantly maintaining and you want to let other people publish new versions without needing your permission.
